src/fvOptions/actuatorLineSource/actuatorLineElement/actuatorLineElement.C

```diff
@@ -477,7 +482,8 @@ void Foam::fv::actuatorLineElement::createOutputFile()
     outputFile_ = new OFstream(dir/name_ + ".csv");
 
     *outputFile_<< "time,root_dist,x,y,z,rel_vel_mag,Re,alpha_deg,"
-                << "alpha_geom_deg,cl,cd,fx,fy,fz,end_effect_factor" << endl;
+                << "alpha_geom_deg,cl,cd,fx,fy,fz,end_effect_factor,"
+                << "Ctan,Cth" << endl;
```

Comment:
Theses names should be all lowercase for consistency. Something like `c_ref_t` and `c_ref_n` for tangential reference and normal reference coefficients may be good (`c_ref_t` would be the azimuthal component for an AFT).